bundler/helpers/v2/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -8,11 +8,11 @@ def index
       if ruby_version
         requested_version = ruby_version.to_gem_version_with_patchlevel
         sources.metadata_source.specs <<
-          Gem::Specification.new("ruby\0", requested_version)
+          Gem::Specification.new("Ruby\0", requested_version)
```

Comment:
Looks like it changed here: https://github.com/rubygems/rubygems/commit/4e950f56d61fc0c276651cefe59ccd87b8a465ad